tests/Base/SqlMigrationSchemaTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Rbac\Cycle\Tests\Base;
+
+use DirectoryIterator;
+
+abstract class SqlMigrationSchemaTest extends SqlSchemaTest
```

Comment:
While exist one migration it works, but when will create second migration we must check different DB state for each migration. 

Suggest here check directly `$migrationsFolderPath . '/1'` only.

Reply:
Is this really needed?

Reply:
Incorrect suggestion.